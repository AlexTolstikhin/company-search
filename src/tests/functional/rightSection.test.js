import React from 'react';
import renderer from 'react-test-renderer';
import RightSection from '../../components/functional/rightSection';

it('snaps RightSection with undefined props', () => {
	const tree = renderer
		.create(<RightSection />)
		.toJSON();
	expect(tree).toMatchSnapshot();
});


it('snaps RightSection with defined props', () => {
	const dummyData = {
		FirstURL: 'fuf',
		Text: 'luf'
	}
	const tree = renderer
		.create(<RightSection isSmallScreen results={[dummyData]}/>)
		.toJSON();
	expect(tree).toMatchSnapshot();
});