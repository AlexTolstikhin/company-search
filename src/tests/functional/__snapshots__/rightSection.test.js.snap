// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`snaps RightSection with defined props 1`] = `
<section
  style={
    Object {
      "width": "100%",
    }
  }
>
  <h2>
    Relative Topics
  </h2>
  <ul
    style={
      Object {
        "margin": "10px",
      }
    }
  >
    <li
      style={
        Object {
          "alignItems": "center",
          "background": "#c7ede9",
          "borderRadius": "15px",
          "display": "flex",
          "height": "100px",
          "justifyContent": "center",
          "margin": "10px",
          "padding": "10px",
        }
      }
    >
      <a
        href="fuf"
        target="blank"
      >
        luf
      </a>
    </li>
  </ul>
</section>
`;

exports[`snaps RightSection with undefined props 1`] = `
<section
  style={
    Object {
      "width": "40%",
    }
  }
>
  <h2>
    Relative Topics
  </h2>
  <ul
    style={
      Object {
        "margin": "10px",
      }
    }
  />
</section>
`;
